crates/sui-framework/packages/move-stdlib/sources/type_name.move

```diff
@@ -106,10 +105,10 @@ module std::type_name {
         let mut i = address::length() * 2 + 2;
         let str_bytes = self.name.as_bytes();
         let mut module_name = vector[];
-
+        let colon = ASCII_COLON;
```

Comment:
is this to avoid the `LdConst` for every loop cycle?

Reply:
Correct. Not related to this PR... it just bothered me.